bindata/assets/kube-apiserver/pod.yaml

```diff
@@ -117,12 +117,12 @@ spec:
       periodSeconds: 5
       timeoutSeconds: 10
       successThreshold: 1
-      failureThreshold: 1
+      failureThreshold: 3
```

Comment:
With these settings, we don't need to change the shutdown delay (70s) or the terminationGracePeriodSeconds (135s).